src/components/Avatar/Avatar.scss

```diff
@@ -1,19 +1,18 @@
 @import '../../styles/common';
 
-$extra-small-size: rem(24px);
-$small-size: rem(32px);
-$medium-size: rem(40px);
-$large-size: rem(60px);
-
 .Avatar {
+  --pc-extra-small-size: 24px;
+  --pc-small-size: 32px;
+  --pc-medium-size: 40px;
+  --pc-large-size: 60px;
```

Comment:
These might have some conflicts with @kyledurand's PR. We also need to do `--pc-avatar-` so we don't get any nested problems with similar custom properties e.g. `--pc-small-size` for avatar and `--pc-small-size` for text inside it.